dali/test/operator_argument.h

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Maybe just use boost::any as we already use small parts of boost, and this is something that is available in later standards.

Reply:
...as a matter of fact we talked about it with @JanuszL today. I implemented `any` (for pre-c++17) for use in Kernels because we didn't want to use boost in the API. When we have it, we can just as well use it here...

Reply:
...still, using these helpers serves a different purpose - it decouples quirks of DALI operators (such as mapping integral types to int64) from testing framework, which should not be aware of them.